docs/app/components/component-section/component-section.component.html

```diff
@@ -15,17 +15,8 @@ <h3 class="uxd-component-section-title">{{ title }}</h3>
             <!-- Show usage link if required -->
             <uxd-usage-link *ngIf="usage" [usage]="usage"></uxd-usage-link>
 
-            <!-- Show the Angular Version -->
-            <span
-                *ngIf="version === 'AngularJS'"
-                class="label label-primary uxd-component-section-labels"
-                >AngularJS</span
-            >
-            <span
-                *ngIf="version === 'Angular'"
-                class="label label-accent uxd-component-section-labels"
-                >Angular</span
-            >
+            <!-- Show UAU migration if available -->
```

Comment:
This is completely subjective on my part, but I find this acronym absolutely ugly so please avoid it. You can shorten the project name to Universal in informal settings.